doc/examples/bundle_shape_similarity.py

```diff
@@ -0,0 +1,111 @@
+"""
+==================================
+BUAN Bundle Shape Similarity Score
+==================================
+
+This example explains how we can use BUAN [Chandio2020]_ to calculate shape
+similarity between two given bundles. Where, shape similarity score of 1 means
+two bundles are extremely close in shape and 0 implies no shape similarity
+whatsoever.
+
+Shape similarity score can be used as a quality assurance metric.
+
+It can be used to validate streamline registration quality, bundle extraction
+quality by calculating output with a reference bundle or other issues with
+pre-processing by calculating shape dissimilarity with a reference bundle.
```

Comment:
```suggestion
Shape similarity score can be used to compare populations or individuals. 
It can also as a quality assurance metric, to validate streamline registration 
quality, bundle extraction quality by calculating output with a reference bundle 
or other issues with pre-processing by calculating shape dissimilarity with a 
reference bundle.
```